app/controllers/api/v1/employees_controller.rb

```diff
@@ -35,8 +35,9 @@ def start_on_dates
 
         dates_rates_hash = rates_for?(start_on..end_on)
         dates = dates_rates_hash.collect { |k, v| k.to_date.to_s.gsub!('-', '/') if v }.compact
+        is_display_sic_codes_enabled = QuotingToolRegistry.feature_enabled?(:display_sic_codes)
 
-        render json: { dates: dates, is_late_rate: !dates_rates_hash.values.all? }
+        render json: { dates: dates, is_late_rate: !dates_rates_hash.values.all?, is_display_sic_codes_enabled: is_display_sic_codes_enabled }
```

Comment:
can you add a spec expecting this in response!

Reply:
Done 👍